addons/markers/functions/fnc_initInsertMarker.sqf

```diff
@@ -93,17 +90,9 @@
     _display displayAddEventHandler ["KeyDown", {(_this select 1) in [200, 208]}];
 
     private _hasTimestamp = false;
-    if !((markerText GVAR(editingMarker)) isEqualTo "") then {
+    if !(markerText GVAR(editingMarker) isEqualTo "") then {
         // fill text input with text from marker which is being edited
-
-        private _originalText = markerText GVAR(editingMarker);
-        private _timeIndex = _originalText find (TIMESTAMP_SPACE + "[");
-        if (_timeIndex > 0 ) then {
-            // Shave off timestamp
-            _hasTimestamp = true;
-            _originalText = _originalText select [0,_timeIndex];
-        };
-
+        private _originalText = markerText GVAR(editingMarker) call FUNC(removeTimestamp);
```

Comment:
Calling `markerText` twice.

Reply:
The trade off is a variable. And that variable is only used once for every marker creation, and only the minimum two for the arguably much rarer marker editing. I think both methods are equal.

Reply:
This line and the `if` statement above would become more readable. Also single source of value is always better.

Reply:
markerText is much faster in 2.02 than it was before.